common/test/db_cache/bok_choy_schema_default.sql

```diff
@@ -3155,8 +3155,6 @@ CREATE TABLE `programs_programsapiconfig` (
   `enable_student_dashboard` tinyint(1) NOT NULL,
   `changed_by_id` int(11) DEFAULT NULL,
   `cache_ttl` int(10) unsigned NOT NULL,
-  `authoring_app_css_path` varchar(255) NOT NULL,
-  `authoring_app_js_path` varchar(255) NOT NULL,
```

Comment:
I don't think you can make these changes until the fields are actually removed from the model. Until that happens, Django's expecting the columns to be present.